### DISCUSSION TITLE
Locked out of my account, support page doesn't work, I can't contact anyone about my issue, email validation needs work

### DISCUSSION THREAD

**Discussion by danny-avila** [community/community#128413] [npm]
I accidentally signed up with an invalid email address and I'm really paying the price with this broken support system.

It took me 5 different browsers to finally get the captcha loading on https://www.npmjs.com/support?inquire=account. Firefox, google chrome, mobile, edge then edge with private window. 

Now that i can play the 5 games, I get an "error loading page" banner for an unsuccessful submission of my "contact us" ticket. Before this, I was getting "unsuccessful captcha" messages.

I signed up with email@gmail instead of email@gmail.com, and now I'm locked out of the username I tried signing up with.

Hopefully this can be resolved as soon as possible, and maybe there needs to be better email validation, even my browser autocorrect has better validation and will tell me it's an invalid email address.

I tried a manual email, and this is what I got:

> Hi there,
> We now require that new support requests be created using our Support website:
> https://www.npmjs.com/support
> 
> Please use this website to search through resources that may help you find the solutions you are looking for or connect with our GitHub experts.
> 
> Thank you,
> GitHub Support
> 
> **Please do not reply to this email, as it will not be seen by our team.**

**Comment by Sleepycat-SAP** [comment]
Always got " error loading page" in this page. https://www.npmjs.com/support
How can I raise my issue??????

**Reply by danny-avila** [reply]
I see I'm not the only one, I think we just have to wait till someone notices this post

**Reply by lchzh3473** [reply]
I just have the same problem now.
![image](https://github.com/npm/feedback/assets/61674328/964d9bce-3be6-4c8b-9cd3-b8dccc685b58)

**Reply by shailen-naidoo** [reply]
I am trying to reach support in regard to being locked out of my account but nobody has gotten back to me about this, what do I do?

**Reply by MrJmpl3** [reply]
I tried to change my email but I have error too, and support don't work

**Reply by shailen-naidoo** [reply]
They ended up resolving my problem
